python/pyspark/pandas/groupby.py

```diff
@@ -308,6 +308,7 @@ def aggregate(
             )
 
         if not self._as_index:
+            index_cols = list(psdf.columns)
```

Comment:
Recommend to use `psdf._internal.data_spark_column_names` here

Reply:
Hello, @itholic. Thank you for the inputs!
I have updated the code but used psdf._internal.column_labels instead as it gave the desired list of tuples. I hope that is fine. Kindly, check.

Reply:
Sounds good. Thanks for addressing